how-to.md

```diff
@@ -32,8 +32,9 @@ Please follow these instructions before your onboarding:
 
 ## Make sure a tenant meets the prerequisites to use Simeon
 
-- You must be operating on global Azure cloud (not [Government Community Cloud](https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/gcc)). The Azure Government cloud is not currently supported
-- Make sure the tenant has a valid Microsoft 365 license - all licensed SKUs are supported
+* Your tenant must be operating on global Azure or the [Government Community Cloud (GCC)](https://learn.microsoft.com/en-us/azure/azure-government/documentation-government-welcome)
+  * Simeon does not support [GCC High and DoD](https://learn.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/gcc-high-and-dod) environments
```

Comment:
GCC High OR DoD

Reply:
MSFT refers to it as "GCC High and DoD", which I think is the name of the product. Referencing it from here: https://learn.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/gcc-high-and-dod